src/containers/profile/profileTab/profileTab.jsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState } from 'react';
+import styles from './styles.module.scss';
+import { FaFacebookF, FaLinkedinIn, FaTwitter} from "react-icons/fa";
+
+export default function profileTab() {
+  const [update, setUpdate] = useState(true);
+
+  return ( 
+    <section className={styles.profile}>
```

Comment:
Use functionality of ES6 like a destructing.

`const {className1, className2}  = styles;`